.github/workflows/lint-and-tests.yml

```diff
@@ -189,7 +190,7 @@ jobs:
         # that changes to dependencies or translations don't break the Unit Tests
         # We want to enforce that the actual `turbo@latest` package is used instead of a possible hijack from the user
         # the `${{ needs.base.outputs.turbo_args }}` is a string substitution happening from the base job
-        run: npx --package=turbo@latest -- turbo test:unit ${{ needs.base.outputs.turbo_args }} -- --ci --coverage
+        run: NODE_OPTIONS="--test-reporter=lcov --test-reporter-destination=lcov.info --test-reporter=junit --test-reporter-destination=junit.xml --test-reporter=@reporters/github --test-reporter-destination=stdout" npx --package=turbo@latest -- turbo test:unit ${{ needs.base.outputs.turbo_args }}
```

Comment:
use `npm run` or `node --run` to reduce de-dupe command. i'm +1 to use `scripts` instead of inline command